src/xml.tsx

```diff
@@ -485,6 +436,16 @@ export function parse(source: string, middleware?: Middleware): JsxAST | null {
     return name;
   }
 
+  function getTag(
+    name: keyof typeof tags
+  ): ComponentType<{ children?: React.ReactNode }> {
+    type Element = (typeof tags)[keyof typeof tags];
+    const Tag: Element | undefined = tags[name];
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    return Tag || missingTag;
```

Comment:
That TS error is valid and should not be muted. We should correct return type of `getTag` function instead.

Reply:
fixed.